packages/toolkit/src/util.ts

```diff
@@ -10,14 +10,11 @@ interface ContractArtifacts {
 }
 
 /**
- * Finds an arapp.json or truffle config file path moving directories recursively up
+ * Finds an arapp.json file path moving directories recursively up
  * @param {boolean} noThrow Whether to throw or not if the root path isn't found
  */
 export function findProjectRoot(noThrow = false): string {
```

Comment:
We should prevent stay coupled to a `node.js` environment to support web compatibility.